### PR TITLE
Feat/metric collection and submission to proxy

### PR DESCRIPTION
**Pull request by vic-en**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

**Tests performed by the developer**:

**Tips for QA testing**:

### CODE REVIEW

File: gateway/src/chains/ethereum/ethereum.ts [hunk 1]
```diff
@@ -85,12 +85,13 @@ export class Ethereum extends EthereumBase implements Ethereumish {
...
       this.requestCount +
-        ' request(s) sent in last ' +
-        this.metricsLogInterval / 1000 +
-        ' seconds.'
-    );
+      ' request(s) sent in last ' +
+      this.metricsLogInterval / 1000 +
+      ' seconds.';
+    logger.info(metricFormat);
+    logger.http(metricFormat);
```
Comment (martinkou) [inline gateway/src/chains/ethereum/ethereum.ts:94]: This feels like a hack to me. In Datadog, metrics are separated from logs - they are not the same entities. Do we have separate endpoints / functionalities in the reporting proxy API for logs and metrics?

**Review by martinkou — CHANGES_REQUESTED** [review]

**Comment by vic-en** [comment]
Notion page for ongoing discussion with Michael Chuang - https://www.notion.so/hummingbot/Requirements-for-Gateway-Datadog-usage-404c7391c9a446539688ff5a645c97f5

File: gateway/src/services/logger.ts [hunk 2]
```diff
@@ -68,10 +69,23 @@ const toStdout = new winston.transports.Console({
   format: sdtoutFormat,
 });
 
+const reportingProxy = new TelemetryTransport({
+  host: 'https://api.coinalpha.com',
```
Comment (martinkou) [inline gateway/src/services/logger.ts:73]: Please don't add "https://" to the host field. Refer to the URI standard.

https://en.wikipedia.org/wiki/Uniform_Resource_Identifier

**Review by martinkou — CHANGES_REQUESTED** [review]

**Comment by mifeng** [comment]
Closing in favor of https://github.com/CoinAlpha/hummingbot/pull/10

**Pull request closed without merging**